### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by youssef-mostafa25**

### CODE REVIEW

**Comment by seifhussam** [comment]
Could you add a table of content tonthe readme, and check to add pull request template to github?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Before submitting this PR make sure:
+
+- Your build is not broken.
+- Your code does exactlt what it is supposed to do
```
Comment (seifhussam) [inline .github/pull_request_template.md]:
```suggestion
- [ ]Your code does exactly what it is supposed to do
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+### Before submitting this PR make sure:
+
+- Your build is not broken.
```
Comment (seifhussam) [inline .github/pull_request_template.md]:
```suggestion
- [ ] build command ran successfully
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+### Before submitting this PR make sure:
```
Comment (seifhussam) [inline .github/pull_request_template.md:1]: Add like a PR description and another heading that contains a bullet list of changes made

**Comment by youssef-mostafa25** [comment]
> Could you add a table of content tonthe readme, and check to add pull request template to github?

done

**Comment by youssef-mostafa25** [comment]
comments done

**Review by seifhussam — APPROVED** [review]
💯

**Pull request merged**